lib/github.ml

```diff
@@ -221,36 +208,39 @@ let run_with_auth ~dry_run auth curl k =
   Sos.run_io ~dry_run curl (OS.Cmd.in_string auth) k
 
 let curl_create_release ~token ~dry_run curl version msg user repo =
-  let parse_release_id resp =
-    (* FIXME this is retired. *)
-    let headers = String.cuts ~sep:"\r\n" resp in
-    try
-      let not_slash c = not (Char.equal '/' c) in
-      let loc = List.find (String.is_prefix ~affix:"Location:") headers in
-      let id = String.take ~rev:true ~sat:not_slash loc in
-      match String.to_int id with
-      | Some id -> Ok id
-      | None ->
-          R.error_msgf "Could not parse id from location header %S: %S" loc id
-    with Not_found ->
-      R.error_msgf "Could not find release id in response:\n%s."
-        (String.concat ~sep:"\n" headers)
+  let parse_release_id json =
+    match Yojson.Basic.Util.member "Location" json with
```

Comment:
There's no such field in the json response, I believe this is part of the response headers which aren't json encoded.

Reply:
Also we should probably get rid of that code and actually parse the id from the JSON encoded response.

Reply:
I changed that but I didn't manage to find a full specification for the response format I hope this field is equivalent to the Location